sys/net/gnrc/application_layer/uhcpc/gnrc_uhcpc.c

```diff
@@ -54,6 +58,47 @@ static void set_interface_roles(void)
 
 static ipv6_addr_t _prefix;
 
+#ifdef MODULE_GNRC_SIXLOWPAN_CTX
+#define SIXLO_CTX_LTIME (60)
```

Comment:
Nitpick: Maybe add something here (either in the macro name or in a comment) that this is a lifetime in minutes

And does it make sense for the end-developer to override this value?

Reply:
We would need to expose this somewhere. Not sure that is worth the effort, as I plan to slowly move over to DHCPv6 ;-).